docker-compose.dev.yml

```diff
@@ -8,6 +8,7 @@ services:
             - mock-sirius-integration
         volumes:
             - ./api/tests/Behat:/var/www/tests/Behat
+            - ./api/src:/var/www/src
```

Comment:
Has this impacted run time at all? I've been finding it super annoying not having the volume and doing manual builds so very happy if performance is still good!

Reply:
Didn't seem to make any difference when running individual tags. I can do a timing test tomorrow if you're uncomfortable about merging before checking that but yeah I didn't realise it was a performance thing. I thought we'd just forgotten it ;)

Reply:
Ok I did have a quick check this morning and it's 251 secs with it and 229 secs without it for a full run. As it's not much difference and locally we would normally be devving and debugging that seems totally worth it for me to not have to rebuild and scratch your head when stuff hasn't changed which you believe should have! ;)

Reply:
Yep agree, lets keep it.